lib/documentmanager.js

```diff
@@ -398,12 +416,45 @@
     };
 
     /**
+     * Re-initialize the current document completely and remove the cached version of all other documents so they'll be
+     * fetched next time they're needed
+     *
+     * @private
+     * @return {Promise} resolved when active document is refreshed
+     */
+    DocumentManager.prototype._resetAllDocuments = function () {
+        var docPromise;
+        if (this._activeDocumentId) {
+            docPromise = this._initDocument(this._activeDocumentId).promise;
+        } else {
+            docPromise = Q(true);
```

Comment:
Is this value `true` needed? In the other case, we return the promise from `_initDocument` which resolves with a `Document`.